apps/aecore/src/aec_peers.erl

```diff
@@ -1579,10 +1579,6 @@ unblock_interval() ->
     application:get_env(aecore, peer_unblock_interval,
                         ?DEFAULT_UNBLOCK_INTERVAL).
 
-
-pool_config() ->
```

Comment:
Why remove this? It is very useful and probably even used for testing

Reply:
`aec_peers_pool:new` now doesn't accept any param, the environment params are read inside the `new/0` function. Tests are adapted to this change.